src/MarginModels/PointSetMarginModel.py

```diff
@@ -65,14 +64,12 @@ def _handleInput(self, paramInput):
     super()._handleInput(paramInput)
     for child in paramInput.subparts:
       if child.getName() == 'failedDataFileID':
-        self.failedDataFileID = self.setVariable(child.value)
-        #self._variableDict['failedDataFileID'] = self.failedDataFileID
+        self.setVariable('failedDataFileID', child.value)
```

Comment:
My suggestion is do not use setVariable here, use the following directly: 
```python 
self.failedDataFileID = child.value
```

Reply:
sorry, i forgot to edit this portion of the code